TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Agreed, I've got a work in progress at the moment, defining the template and starting the Microsoft.NETCore.props/Microsoft.NETCore.targets, which will implicitly be included by Microsoft.CSharp.targets/Microsoft.Common.props.